contracts/stellar-interchain-token-service/src/token_manager.rs

```diff
@@ -10,6 +13,14 @@ pub trait TokenManagerClientExt {
 }
 
 impl TokenManagerClientExt for TokenManagerClient<'_> {
+    fn transfer_token_admin(&self, env: &Env, token_address: &Address, new_admin: &Address) {
```

Comment:
```suggestion
    fn set_admin(&self, env: &Env, token_address: &Address, new_admin: &Address) {
```
name the method after the underlying method that's actually being called for clarity

Reply:
make sense, updated.